views/js/controller/Delivery/monitoring.js

```diff
@@ -873,6 +873,15 @@ define([
                         });
                     }
 
+                    model.push({
+                        id: 'extendedTime',
+                        label: __('Extended Time'),
+                        transform: function(value, row) {
+                            var timer = _.isObject(row.timer) ? row.timer : {};
+                            return timer.extendedTime;
```

Comment:
As the extended time is a factor, maybe should it be better to illustrate it. For instance by prefixing the displayed value with a `x`. This way it could be: `x1.5`. What do you think?